### ISSUE TITLE
Add specification of the start event in trigger instance

### ISSUE THREAD

**Issue by jblankendaal** [effektif/effektif#27]
When a workflow has several start events, they will all be triggered when the flow is started, according to bpmn standard.
So when a flow has two startevents/activities, in my opinion two workflowInstances should be created when the workflow is triggered (For example via the WorkflowEngineImpl.start(TriggerInstance) method.)

However, this is not the case, and I think it 's a bug. What now actually happens is that one workflowInstance is created, and both start events are executed for this instance (ie logged in the activityInstances array). The workflowInstance object is finally advanced to the activity where the last startevent was pointing to.

See the example image below
![twostartevents](https://cloud.githubusercontent.com/assets/11930623/8892861/1ae70d72-3373-11e5-96c5-93dd82f76bb2.png)

In this example after starting the workflow the result is that the flow contains one workflowInstance that resides in the "User 1" activity, while I would expect two workflowInstances: One in "Manual 1" and one in "User 1".

**Comment by jblankendaal** [comment]
Learned a lot :-) 
Turns out that my "Manual 1" activity type was not recognised during deploy. So for the engine the operation did not exist (I think a warning/error should be issued in that case, I will open another issue for that).
In the end both Start events are fired, but with one workflowInstanceId, while I would expect that two workflowInstances were created: one for each Start Event. On this one workflowInstanceId all activityInstances are logged, making it not very clear what flow path was followed, because these paths were followed in parallel.
But: In the end everything works, although a little bit different than I expected.

**Comment by tombaeyens** [comment]
When there are multiple start events, starting all of them at the same time seems not desirable.  Users typically want to start only 1 start event.  The current logic (of starting all activities without incoming transitions) is based on the case where there are no start events and only have tasks / activities in your diagram.

I think that we should extend the TriggerInstance so that 1 event can be fired in 1 workflow instance.

The TriggerInstance should get a field called 'startActivityIds'.  In that field, the client can specify which activities should be started when the workflow instance starts.  A fluent convenience method `startActivityId(String)` should be added to the TriggerInstance that adds a startActivityId to the list.

**Comment by jblankendaal** [comment]
I have already implemented this, and then I ran into the above issues. So if we agree that this is the way to go, I will write a test for it this afternoon and create a pull request.
